sims/verilator/Makefile

```diff
@@ -1,112 +1,207 @@
-#########################################################################################
+#############################################################################
```

Comment:
There are just too many formatting changes to give this file the detailed review it needs. Many of these flags are not exhaustively tested by CI, so we need to be pretty careful about what we change.

Reply:
the solution sounds like beefing up the regression tests. i will do that so we can be confident that formatting changes are not causing regressions.

Reply:
No, the right way to do this is to separate the refactor into multiple PRs.